code/modules/reagents/reagent_containers/medigel.dm

```diff
@@ -87,12 +87,29 @@
 
 /obj/item/reagent_containers/medigel/synthflesh
 	name = "medical gel (synthflesh)"
-	desc = "A medical gel applicator bottle, designed for precision application, with an unscrewable cap. This one contains synthflesh, a slightly toxic medicine capable of healing both bruises and burns."
+	desc = "A medical gel applicator bottle, designed for precision application, with an unscrewable cap. This one contains synthflesh, a slightly toxic medicine capable of healing bruises, burns, and husks."
 	icon_state = "synthgel"
 	current_skin = "synthgel"
 	list_reagents = list(/datum/reagent/medicine/c2/synthflesh = 60)
+	list_reagents_purity = 1
+	amount_per_transfer_from_this = 60
+	possible_transfer_amounts = list(5, 10, 60)
 	custom_price = PAYCHECK_CREW * 5
 
+/obj/item/reagent_containers/medigel/synthflesh/examine(mob/user)
+	. = ..()
+	if(reagents.total_volume >= 60)
+		. += span_info("One full bottle can restore a corpse husked by burns.")
+
+/obj/item/reagent_containers/medigel/synthflesh/attack(mob/M, mob/user, def_zone)
+	if(iscarbon(M))
+		var/mob/living/carbon/carbies = M
+		if(HAS_TRAIT_FROM(carbies, TRAIT_HUSK, BURN) && carbies.getFireLoss() > UNHUSK_DAMAGE_THRESHOLD)
```

Comment:
This threshold is a little misleading. Since the Synthflesh will heal upwards of 70 burn on its own, the threshold is closer to `UNHUSK_DAMAGE_THRESHOLD * 2`. 

(In the case of the patch) I'd get annoyed if the application delay was increased even though I knew what I was doing

Reply:
I can remove the delay, the thought was to give them a moment to read the message. But yeah in that case I see it being annoying.

Reply:
> This threshold is a little misleading. Since the Synthflesh will heal upwards of 70 burn on its own, the threshold is closer to `UNHUSK_DAMAGE_THRESHOLD * 2` - `UNHUSK_DAMAGE_THRESHOLD * 2.25`.

Yeah this bugs me, I'm going to find the correct calculation and put it in there.